lib/Account.php

```diff
@@ -173,4 +185,45 @@ public static function createLoginLink($id, $params = null, $opts = null)
     {
         return self::_createNestedResource($id, static::PATH_LOGIN_LINKS, $params, $opts);
     }
+
+    public function serializeParameters($force = false)
```

Comment:
Just OOC, is this mostly copied from the Ruby implementation? The test suite looks like it's pretty good in terms of vetting the new code, but just wondering how much we should expect to have to worry about it.

Reply:
Yep, it's basically a straight port from the Ruby implementation.